analysis/tbcpriest/src/modules/features/HealingEfficiencyBreakdown.tsx

```diff
@@ -0,0 +1,80 @@
+import BaseHealingEfficiencyBreakdown from 'parser/core/healingEfficiency/HealingEfficiencyBreakdown';
+import React from 'react';
+import Toggle from 'react-toggle';
+
+class HealingEfficiencyBreakdown extends BaseHealingEfficiencyBreakdown {
```

Comment:
What did you change here from the base? Is this not already in the base?

Reply:
I removed the duplicate code.